### ISSUE TITLE
Difference between old and slim inception v3 models

### ISSUE THREAD

**Issue by nrupatunga** [tensorflow/models#879]
I  retrained inception V3 model using the old inception V3 scripts available in (https://github.com/tensorflow/models/tree/master/inception) for the classification task on my dataset. 
Later I used the updated TF-slim version of Inception V3 (https://github.com/tensorflow/models/tree/master/slim) to retrain the model. 

But, what I observed was, the inception V3 available in slim code didn't converge well when compared to old inception-v3. 

1. One issue was in the slim data preparation did not shuffle the given data while preparing the tf-record. This I resolved. 
2. After resolving the issue 1, I found that the loss were in sync with old inception-v3. but the slim based inception v3 didn't converge well.

Old inceptionv3 model gave accuracy above 80%
While slim inception v3 gave an accuracy of 55% 

3. I have checked the training script, even though different hyper parameter settings were used(like moving average, label loss regularizer), making hyperparameters same also didn't help. 

Can someone give some pointers on what is the difference in these two models?

**Comment by nrupatunga** [comment]
For whoever dealing with issue. 
please check the data preparation, preprocessing part and hyper parameter settings in order to resolve the issue. 

Issue is resolved so closing
